react/components/InfoCard/index.js

```diff
@@ -95,6 +99,10 @@ const InfoCard = ({
   htmlId,
   textMode,
   linkTarget,
+  analyticsProperties = 'none',
+  promotionId,
+  promotionName,
+  promotionPosition,
```

Comment:
Since these props would only be useful if `analyticsProperties` is set to something different than `none`, why not make `analyticsProperties` an object with these props? Then its default value would be `undefined`.

Reply:
Hey @victorhmp, really sorry for the delay, I've had some big demands here.

As stated in the PR, this new feature was implemented from store-image component and in this component these props are declared in the same mode. I tried to change they as your request, but I couldn't do this without changing the admin,   noticing some problems in site-editor.

Reply:
@victorhmp

Reply:
Add these properties on propTypes.